code/game/world.dm

```diff
@@ -120,9 +120,15 @@ GLOBAL_LIST(topic_status_cache)
 	GLOB.world_runtime_log = "[GLOB.log_directory]/runtime.log"
 	GLOB.query_debug_log = "[GLOB.log_directory]/query_debug.log"
 	GLOB.world_job_debug_log = "[GLOB.log_directory]/job_debug.log"
+<<<<<<< HEAD
```

Comment:
uhhhhhhhhhhhhhhhhhhhhhhhh

Reply:
This happens when the mirrorbot gets merge conflicts. Atleast it force-fails travis so we know it dun goofed